### DISCUSSION TITLE
Unable to change global border color, no matter what I try

### DISCUSSION THREAD

**Discussion by ilmoi** [chakra-ui/chakra-ui#6237] [General] [answered]
Can someone with more chakra knowledge please advise on this issue? https://stackoverflow.com/questions/72738377/chakra-cant-override-global-border-style

**Comment by Mayvis** [comment, marked as answer]
@ilmoi Have you tried added '*' style borderColor?

```jsx
const theme = extendTheme({
  styles: {
    global: {
      '*': {
        borderColor: 'black !important'
      }
    }
  }
})
```
